app/coffeescripts/views/assignments/EditView.coffee

```diff
@@ -193,6 +201,14 @@ AssignmentGroupSelector, GroupCategorySelector, toggleAccessibly, RCEKeyboardSho
         turnitinDialog.off()
         turnitinDialog.remove()
 
+    showVeriCiteDialog: (ev) =>
+      ev.preventDefault()
+      vericiteDialog = new VeriCiteSettingsDialog(model: @assignment.get('vericite_settings'))
```

Comment:
I'm struggling trying to figure out how to combine these two since I need VeriCite and Turnitin to have separate settings. This includes: "VeriCiteSettingsDialog" and "vericite_settings". Any suggestions?